assets/src/application/services/theme/ThemeProvider.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import { ThemeProvider as ChakraThemeProvider } from '@chakra-ui/core';
+import { ThemeProvider as ChakraThemeProvider, CSSReset } from '@chakra-ui/core';
 
 import theme from './theme';
 
 const ThemeProvider: React.FC = ({ children }) => {
-	return <ChakraThemeProvider theme={theme}>{children}</ChakraThemeProvider>;
+	return (
+		<ChakraThemeProvider theme={theme}>
+			<CSSReset />
+			{children}
+		</ChakraThemeProvider>
+	);
```

Comment:
Enabled `CSSReset`, more about it [here](https://chakra-ui.com/getting-started#injecting-global-styles)